13 jan/center-text.js

```diff
@@ -0,0 +1,36 @@
+const BLANK_SPACE = ' ';
+const EMPTY_STRING = ' ';
+
+var alignmentUtil = {
```

Comment:
There is no need of creating an object. Just the method `expandString` would also work.

Reply:
This way I am making the alignment Util extendable.